cms/static/cms/js/build.js

```diff
@@ -171,5 +171,4 @@
          */
         logLevel: 1
     };
-}())  // eslint-disable-line semi
-// A semicolon on the line above will break the requirejs optimizer
+}());
```

Comment:
@bjacobel or @andy-armstrong - this removal of a hacky situation appears to be necessary to allow for the upgraded version of `django-require` contained in this PR - can you verify that this is a sane change?